docs/usage.md

```diff
@@ -116,14 +117,21 @@ ChIPSampleID3,,AnalysisID3
 For single-sample peaking calling without a control sample, leave the second column blank
 (control sample name).
 
+### `--macsgsize`
+Effective genome size which is used for the option `--gsize` in MACS. Should be in the format "2.1e9". See [`conf/igenomes.config`](conf/igenomes.config) for the predefined values of all supported reference genomes.
+
 ### `--broad`
 Run MACS with the `--broad` flag. With this flag on, MACS will try to composite broad regions in BED12 ( a gene-model-like format ) by putting nearby highly enriched regions into a broad region with loose cutoff. The broad region is controlled by the default qvalue cutoff 0.1.
 
 ### `--saturation`
 Run saturation analysis by sub-sampling the sequence reads of ChIP sample from 10% to 100% with 10% interval, then calling ChIP-seq peaks with MACS. For one test there will be 10 sets of ChIP-seq peaks. A summary file (.CSV) will be provided with the numbers of ChIP-seq peaks.
 
 ### `--blacklist_filtering`
-Specifying this flag instructs the pipeline to use bundled ENCODE blacklist regions to filter out known blacklisted regions in the called ChIP-seq peaks. Please note that this is only supported when --genome is set to `GRCh37` or `GRCm38`.
+Specifying this flag instructs the pipeline to use bundled ENCODE blacklist regions to filter out known blacklisted regions in the called ChIP-seq peaks. The following reference genome builds are supported:
+* Human: `GRCh37`, `GRCh38`
+* Mouse: `GRCm37`, `GRCm38`
+* C.elegans: `WBcel235`
+* D.melanogaster: `BDGP6`
```

Comment:
Would be good to describe what should be supplied with this flag. eg. Provide a path to a `BED` file containing regions you would like to be ignored. If not supplied, this step will be skipped.

Reply:
It is already included in the section of `--blacklist`

Reply:
Ah sorry, I misread this. But can this not be included in the `--genome` flag automatically? Why do we need to flag it twice?

Reply:
Because only 4 genomes have precompiled blacklist files.